aten/src/ATen/native/Sorting.cpp

```diff
@@ -708,5 +708,20 @@ std::tuple<Tensor, Tensor> sort_cpu(
   return sort_out_cpu(values, indices, self, dim, descending);
 }
 
+std::tuple<Tensor&, Tensor&> msort_out(
+        Tensor& values,
+        Tensor& indices,
+        const Tensor& self,
+        bool descending) {
+  at::sort_out(values, indices, self, 0, descending);
+  return std::forward_as_tuple(values, indices);
+}
+
+std::tuple<Tensor, Tensor> msort(
+        const Tensor& self,
+        bool descending) {
```

Comment:
Let's not implement the `descending` argument for now. NumPy's msort doesn't have one.

Reply:
Addressed!